### PR TITLE
Update text

### PR DESCRIPTION
**Pull request by jkrick**
Polishing....
I changed some things to markdown header, edited lots of text, added sections (questions) and added some images to guide the users.

Links to places within the markdown are not working (and I am stumped).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,192 +1,213 @@
...
+  * In Jupyter, kernels are the background processes that execute cells and return results for display.
+  * To select the kernel on which you want to run your Notebook, go to the Kernel menu and choose Change Kernel. You can also click directly on the name of the active kernel to switch to another.
+    
+## Navigating jupyter lab
+### How do I start a new notebook?
+  * The blue `+` in the upper left brings you to the launcher where you can start a new, empty notebook or open a terminal window
+  * * ![new launcher](./new_launcher.png)
+### How do I get a terminal window?
+  * The blue `+` in the upper left brings you to the launcher where you can start a new notebook or open a terminal window
+  * ![terminal](./terminal.png)
```
Comment (bsipocz) [inline README.md:64]: maybe merge these two, e.g. `### How to start a new notebook / get a new window`? the text is shared but the screenshots are not

File: README.md [hunk 2]
```diff
@@ -1,192 +1,213 @@
...
+### How to upload data into the Fornax Science Console?
+  * The `uparrow` in the upper left allows you to upload data
+  * ![upload_button](./upload_button.png)
+  * if it is a large amount of data, consider creating a zip or tar archive first
+  * From within Jupyter Lab, you can also use the shell to 'scp' or 'git pull' from external sites that are publicly visible.
```
Comment (bsipocz) [inline README.md:69]: not sure what you mean by `git pull`, maybe rephrase?

```suggestion
  * From within Jupyter Lab, you can also use a terminal to transfer data with the usual methods (`scp`, `wget`, `curl` should all work).
```

File: README.md [hunk 3]
```diff
@@ -1,192 +1,213 @@
...
+    * As far as I can tell, the best (AWS) option for actual controls -- beyond just "monitoring" support -- is [AWS Budgets Actions](https://aws.amazon.com/blogs/aws-cloud-financial-management/get-started-with-aws-budgets-actions/).
+    * AWS may charge *ingress* fees to bring data into an SMCE pod or user instance. This would be completely separate from any egress fees. There Someone working more directly on Fornax Daskhub would need to answer whether/how ingress applies.
+### The SP will be most beneficial to use cases which:
```
Comment (bsipocz) [inline README.md:32]: all usage of `SP` will need to be reviewed and probably changed to `Fornax Console`?

**Review by bsipocz — APPROVED** [review]
some comments but none are critical, I spotted a few inconsistencies and formatting issues.

**Comment by bsipocz** [comment]
Would you find it useful to have a preview link in the PRs like we have for the navo notebooks?

**Comment by jkrick** [comment]
> Would you find it useful to have a preview link in the PRs like we have for the navo notebooks?

I'm not sure what you mean here @bsipocz .  I think I managed fine with this PR to do edits and comments on the readme.md
